core/server/models/post.js

```diff
@@ -163,6 +163,21 @@ Post = ghostBookshelf.Model.extend({
             tagsToSave,
             ops = [];
 
+        // CASE: the client might send empty image properties with "" instead of setting them to null.
+        // This can cause GQL to fail. We therefore enforce 'null' for empty image properties.
+        // See https://github.com/TryGhost/GQL/issues/24
+        if (this.get('feature_image') === '' && this.previous('feature_image')) {
```

Comment:
Can't we use `onValidate`?

`onValidate` is always called after `onSaving`.

Post
```
onValidate
   validate '' fields
   call parent
```

Reply:
Alright. Done that. Also used a generic fn now. Will change the tests now.